src/placeos-driver/proxy/scheduler.cr

```diff
@@ -44,11 +55,11 @@ class PlaceOS::Driver::Proxy::Scheduler
     spawn(same_thread: true) { run_now(block) } if immediate
     wrapped = nil
     task = Tasker.at(time) do
-      @schedules.delete(wrapped.not_nil!.object_id)
+      @schedules.delete(wrapped.not_nil!)
```

Comment:
Any reason to not do something like this to prevent a possible `Nil assertion failed` error?
```suggestion
      @schedules.delete(wrapped) if wrapped
```